application.py

```diff
@@ -16,23 +17,24 @@
 
 
 def configure_logging():
-    # set up some sane logging, as opposed to what flask does by default
     log_format = "%(message)s"
-    levels = {
-        'CRITICAL': logging.CRITICAL,
-        'ERROR': logging.ERROR,
-        'WARNING': logging.WARNING,
-        'INFO': logging.INFO,
-        'DEBUG': logging.DEBUG,
-    }
-    handler = logging.StreamHandler()
+
+    info_handler = logging.StreamHandler(sys.stdout)
+    info_handler.setLevel(logging.getLevelName(EQ_LOG_LEVEL))
```

Comment:
I think we should remove `EQ_LOG_LEVEL` as we use `INFO` everywhere, and turn debug logging on based on the application `DEBUG` mode.